Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
```

Comment:
It will use the latest `3.x.x` version, but probably better to restraint it to `3.10.x` to avoid any compatible issues whenever a newer version is released 

https://docs.docker.com/engine/reference/builder/#official-releases